ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.txt

```diff
@@ -81,7 +81,7 @@ hideall "--sync--"
 #238.3 "Thermionic Beam" Ability { id: "26BD", source: "Nael deus Darnus" }
 266.4 "Doom x2" Ability { id: "26C9", source: "Tail of Darkness" }
 268.4 "Fireball (1)" Ability { id: "26C5", source: "Firehorn" }
-269.4 "Wings Of Salvation x2" Ability { id: "26CA", source: "Fang of Light" } duration 4
+269.4 "Wings of Salvation x2" Ability { id: "26CA", source: "Fang of Light" } duration 4
```

Comment:
Here and elsewhere, could you capitalize "Of"? This is a thing that we should be adding to `make_timeline` already, but even if the game doesn't capitalize prepositions and conjunctions, we still prefer to do so in timeline entries and triggers. (There are definitely situations where we've overlooked it in the repository, but we generally try.) The timeline controller *should* be handling any case issues automagically, so if you are seeing sync issues in German, that's worth filing a bug report.

Reply:
Sofar i did not expirienced any sync issues, just found both translations and picked the wrong one xD
Adjusted the "of" to "Of